src/App.tsx

```diff
@@ -15,71 +15,67 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+export class App extends Component {
+  state = {
+    selected: 'Jam',
+  };
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  handleItemClick = (itemName: string) => {
+    if (itemName === this.state.selected) {
+      this.setState({ selected: '' });
+    } else {
+      this.setState({ selected: itemName });
+    }
+  };
```

Comment:
look how Miłosz sugested it: 
```ts
handleItemClick = (itemName: string) => {
  if (itemName === this.state.selected) {
    this.setState({ selected: '' });
    return;
  };
  this.setState({ selected: itemName });
}
```

You should avoid `else-ing` when possible, and just add a simple early return